apps/frontend/src/app/5_pages/BorrowPage/components/AdjustLoanForm/AdjustLoanForm.tsx

```diff
@@ -674,6 +682,42 @@ export const AdjustLoanForm: FC<AdjustLoanFormProps> = ({ loan }) => {
     [isValidDebtAmount],
   );
 
+  const [drawDown, setDrawDown] = useState(Decimal.ZERO);
```

Comment:
It looks good but pls extract it to a custom hook that accepts `loan`, `isAddCollateralTab` and returns `drawdown` and `maxBorrow`.